template/partial/settings.html

```diff
@@ -0,0 +1,49 @@
+<form class="ui form signup-form">
+	<div class="two fields">
+		<div class="field">
+			<label>Email</label>
+			<input type="text" name="email" id="email" placeholder="Email">
```

Comment:
Name and ID have different usage. Name is used by JS while ID is a style selector. Both can have same values as the selectors are different for them.
- [Name Selector](https://api.jquery.com/attribute-contains-selector/): [name*=”value”]
- [ID Selector](https://api.jquery.com/id-selector/): $(#<id>)

Reference: [Name vs ID](https://stackoverflow.com/questions/7470268/html-input-name-vs-id)